packages/material-ui/src/Menu/Menu.js

```diff
@@ -157,7 +157,10 @@ const Menu = React.forwardRef(function Menu(inProps, ref) {
     <MenuRoot
       classes={PopoverClasses}
       onClose={onClose}
-      anchorOrigin={isRtl ? RTL_ORIGIN : LTR_ORIGIN}
+      anchorOrigin={{
+        vertical: 'bottom',
+        horizontal: isRtl ? 'right' : 'left',
+      }}
       transformOrigin={isRtl ? RTL_ORIGIN : LTR_ORIGIN}
```

Comment:
Should the `transdformOrigin` be changed too?

Reply:
I don't think so. For Menu use cases I see in Google Product and Material Design spec, it appear from the `left` (not `center`).